### ISSUE TITLE
损失函数

### ISSUE THREAD

**Issue by xiayouhong** [GanjinZero/RRHF#42]
您好，损失函数中的 t 是啥意思呀

**Comment by GanjinZero** [comment]
第t个token

**Comment by xiayouhong** [comment]
![image](https://github.com/GanjinZero/RRHF/assets/61572755/71dab144-81ad-4d11-ad67-d75aaabc5c9f)
您好，在logit_label = self.gather_logits_labels(logits, inputs.get("labels"))这一步过程中inputs.get("labels“）会被修改，会影响下一步score的计算

**Comment by GanjinZero** [comment]
https://github.com/GanjinZero/RRHF/issues/37
